mount/losetup_linux_test.go

```diff
@@ -86,7 +86,13 @@ func TestRwLoop(t *testing.T) {
 	if err != nil {
 		t.Fatal(err)
 	}
-	defer file.Close()
+	defer func() {
+		dev := file.Name()
+		file.Close()
+		if err := removeLoop(dev); err != nil {
+			t.Fatal(err)
+		}
+	}()
```

Comment:
Alternatively, for conciseness should the solution be to just use `Autoclear: true` similar to`TestRoLoop`?

Reply:
@austinvazquez  How about setting `Autoclear: ture` in `TestRwLoop`, and add a new test case for `Autoclear: false`?

Reply:
+1, yeah it was not clear before that `TestRwLoop` was intended to test `Autoclear: false`. The tests you have added are much more intentional. Thanks.